pkg/apis/core/v1/helper/helpers.go

```diff
@@ -26,7 +26,9 @@ import (
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/selection"
 	"k8s.io/apimachinery/pkg/util/sets"
+	kubeapi "k8s.io/kubernetes/pkg/apis/core"
 	"k8s.io/kubernetes/pkg/apis/core/helper"
+	schedulerapi "k8s.io/kubernetes/pkg/scheduler/api"
```

Comment:
The direction of this dependency looks backwards.  Seems odd for apis/core/v1 to depend on the scheduler.

Reply:
Thanks for clarification @deads2k. I have moved the fn's around to remove this dependency.